Index: VTK-6.1.0/CMake/FindNetCDF.cmake
===================================================================
--- VTK-6.1.0.orig/CMake/FindNetCDF.cmake	2014-01-22 10:55:41.000000000 -0500
+++ VTK-6.1.0/CMake/FindNetCDF.cmake	2014-04-04 17:34:54.704403018 -0400
@@ -86,7 +86,7 @@
 if (_nextcomp GREATER -1)
   set (NETCDF_F90 1)
 endif ()
-NetCDF_check_interface (CXX netcdfcpp.h netcdf_c++)
+NetCDF_check_interface (CXX netcdf netcdf_c++4)
 NetCDF_check_interface (F77 netcdf.inc  netcdff)
 NetCDF_check_interface (F90 netcdf.mod  netcdff)
 
