Index: VTK-6.1.0/Rendering/OpenGL/vtkXOpenGLRenderWindow.cxx
===================================================================
--- VTK-6.1.0.orig/Rendering/OpenGL/vtkXOpenGLRenderWindow.cxx
+++ VTK-6.1.0/Rendering/OpenGL/vtkXOpenGLRenderWindow.cxx
@@ -27,7 +27,7 @@
 
 // define GLX_GLXEXT_LEGACY to prevent glx.h to include glxext.h provided by
 // the system
-//#define GLX_GLXEXT_LEGACY
+#define GLX_GLXEXT_LEGACY
 #include "GL/glx.h"
 
 #include "vtkgl.h"
